aya/src/sys/mod.rs

```diff
@@ -64,7 +69,7 @@ unsafe fn syscall_impl(call: Syscall) -> SysResult {
             flags,
         } => libc::syscall(SYS_perf_event_open, &attr, pid, cpu, group, flags),
         PerfEventIoctl { fd, request, arg } => {
-            libc::ioctl(fd, request.try_into().unwrap(), arg) as i64
+            libc::ioctl(fd, request.try_into().unwrap(), arg) as RetType
```

Comment:
I think this should be `c_long`

Reply:
Same as before, it was defined as i64, I'll use c_long